the-blue-alliance-ios/View Elements/Match/Breakdown/MatchBreakdownConfigurator2020.swift

```diff
@@ -0,0 +1,258 @@
+        for i in [1, 2, 3] {
+            rows.append(endgameRow(i: i, red: red, blue: blue))
+        }
+        rows.append(shieldSwitchLevelRow(title: "Shield Generator Switch Level", red: red, blue: blue))
+        rows.append(row(title: "Endgame Points", key: "endgamePoints", red: red, blue: blue, type: .subtotal))
+        rows.append(row(title: "Total Teleop", key: "teleopPoints", red: red, blue: blue, type: .total))
+        
+        rows.append(stageActivationRow(title: "Stage Activations", red: red, blue: blue))
+        rows.append(shieldOperationalRow(title: "Shield Generator Operational", red: red, blue: blue))
+        
+        // Match totals
+        rows.append(foulRow(title: "Fouls / Tech Fouls", red: red, blue: blue))
+        rows.append(row(title: "Adjustments", key: "adjustPoints", red: red, blue: blue))
+        rows.append(row(title: "Total Score", key: "totalPoints", red: red, blue: blue, type: .total))
+        
+        // RP
+        rows.append(row(title: "Ranking Points", key: "rp", formatString: "+%@ RP", red: red, blue: blue))
+
+        // Clean up any empty rows
+        let validRows = rows.compactMap({ $0 })
+        if !validRows.isEmpty {
+            snapshot.appendSections([nil])
+            snapshot.appendItems(validRows)
+        }
+    }
+
+    private static func initLine(red: [String: Any]?, blue: [String: Any]?) -> BreakdownRow? {
+        var redLineStrings: [String] = []
+        var blueLineStrings: [String] = []
+        
+        for i in [1, 2, 3] {
+            guard let initValues = values(key: "initLineRobot\(i)", red: red, blue: blue) else {
+                return nil
+            }
+            let (rv, bv) = initValues
+            guard let redInit = rv as? String, let blueInit = bv as? String else {
+                return nil
+            }
+            redLineStrings.append(redInit)
+            blueLineStrings.append(blueInit)
+        }
+        
+        let elements = [redLineStrings, blueLineStrings].map { (lineStrings) -> [AnyHashable] in
+            return lineStrings.map { (line) -> AnyHashable in
+                switch line {
+                case "None":
+                    return BreakdownStyle.xImage
+                case "Exited":
+                    return BreakdownStyle.checkImage
+                default:
+                    return "?"
+                }
+            }
+        }
+        
+        var (redElements, blueElements) = (elements[0], elements[1])
+
+        // Add the point totals for the init line
+        guard let initLinePoints = values(key: "autoInitLinePoints", red: red, blue: blue) else {
+            return nil
+        }
+        
+        let (redLinePoints, blueLinePoints) = initLinePoints
+        redElements.append("(+\(redLinePoints ?? 0))")
+        blueElements.append("(+\(blueLinePoints ?? 0))")
+
+        return BreakdownRow(title: "Initiation Line exited", red: redElements, blue: blueElements)
```

Comment:
We call this `Initiation Line Extended` on web

Reply:
The web was wrong.  😁 

the-blue-alliance/the-blue-alliance#2711